e2e_tests/navigation_journeys/test_search_to_record_journey.py

```diff
@@ -24,7 +24,9 @@ def test_search_to_record(aau_user_page: Page):
 
     aau_user_page.get_by_role("textbox").first.fill("a")
     aau_user_page.get_by_role("button", name="Search").click()
-    expect(aau_user_page).to_have_url("search_results_summary?query=a")
+    expect(aau_user_page).to_have_url(
```

Comment:
so this test is the one about nvaigating to a record i na search journey.

I think the tests in `class TestSearchResultsSummary:` (and possible the other test classes there) in `e2e_tests/test_search.py` would still be broken though, will those be done in a separate PR?

We just need to make sure that we cover the 2 ac in the ticket: https://national-archives.atlassian.net/jira/software/projects/AYR/boards/66?selectedIssue=AYR-1236&text=AYR-1236

The AC do have 2 actions and currently that file splits out these scenarios into 2 test classes with the second starting from the summary results page, but do whatever feels best to you.